src/jupyter_contrib_nbextensions/nbextensions/code_prettify/isort.yaml

```diff
@@ -0,0 +1,45 @@
+Type: Jupyter Notebook Extension
+Name: isort formatter
+Description: Sort imports in python files using isort
+Link: README_isort.md
+Main: isort.js
+Compatibility: Jupyter 4.x, 5.x
+Parameters:
+
+- name: isort.add_toolbar_button
+  description: Add a toolbar button to convert the selected cell(s)
+  input_type: checkbox
+  default: true
+
+- name: isort.button_icon
+  description: |
+    Toolbar button icon: afont-awesome class defining the icon used for the
+    toolbar button. See http://fontawesome.io/icons/ for available icons.
+  input_type: text
+  default: 'fa-sort'
+
+- name: isort.button_label
+  description: Toolbar button label text
+  input_type: text
+  default: 'Sort imports with isort'
+
+- name: isort.show_alerts_for_errors
```

Comment:
since you've caught all exceptions in the library function, this option might be better of not listed, since it won't actually do anything